process/sandbox-annual-review.md

```diff
@@ -0,0 +1,45 @@
+# Sandbox annual review
+
+Sandbox projects are subject to an annual review by the TAC. This is intended to be a lightweight process to ensure that projects are on track, and getting the support they need.
+
+To keep it lightweight, it is a single phase process - the review doesn't first pass through TAGs and then the TAC. The review document is shared with the community and TAG members can comment on each review, as can any other TAC contributor. The intention is to quickly reach a position where TAC members agree to continue sponsorship.
```

Comment:
This refers to "TAGs" which we don't currently have and isn't defined anywhere in this PR that I can see.